contracts/DaiHrd.sol

```diff
@@ -599,12 +590,12 @@ contract DaiHrd is ERC777, MakerFunctions {
 	}
 
 	// Internal implementations of functions with multiple entrypoints. drip() should be called prior to this call
-	function _withdraw(address recipient, uint256 attodaiHrd) internal returns(uint256 attodai) {
-		require(recipient != address(0) && recipient != address(this), "Invalid recipient");
+	function withdraw(address recipient, uint256 attodaiHrd) private returns(uint256 attodaiWithdrawn) {
```

Comment:
This was the only private function with a leading `_` so I removed it for consistency.